doc/source/working_with_kiwi/legacy_kiwi.rst

```diff
@@ -141,15 +142,15 @@ generation KIWI.
    already exists on your system, the alternative setup will skip the
    creation of the link target because it already exists.
 
-From an appliance description perspective, both KIWI versions are fully
+From an appliance description perspective, both versions are fully
 compatible. Users can build their appliances with both versions and the
 same appliance description. If the appliance description uses features
-the next generation KIWI does not provide, the build will fail with an
+the {kiwi-product} does not provide, the build will fail with an
 exception early. If the appliance description uses next generation
 features like the selection of the initrd system, it's not possible to
-build that with the legacy KIWI, unless the appliance description
+build that with the {kiwi-legacy}, unless the appliance description
 properly encapsulates the differences into a profile.
 
-The next generation KIWI also provides the `--compat` option and
+The {kiwi-product} also provides the `--compat` option and
```

Comment:
Again, the product is already introduced so `{kiwi}` should be enough distinction against Legacy KIWI.